app/api/bookyt/entities/base.rb

```diff
@@ -0,0 +1,10 @@
+module Bookyt
+  module Entities
+    class Base < Grape::Entity
+      format_with(:iso_timestamp) do |dt|
+        dt ? dt.iso8601 : nil
+      end
+      format_with(:float, &:to_f)
```

Comment:
Unfortunately JSON does not support `BigDecimal`, how should we handle this? Casting it to a string?